cxx/emissions/get_emission_test.cc

```diff
@@ -0,0 +1,40 @@
+#define BOOST_TEST_MODULE test get_emission
+
+#include "emissions/get_emission.hh"
+#include "emissions/bitflip.hh"
+#include "emissions/gaussian.hh"
+#include "emissions/simple_string.hh"
+
+#include <boost/test/included/unit_test.hpp>
+
+BOOST_AUTO_TEST_CASE(test_get_emission_gaussian) {
+  EmissionVariant ev = get_emission("gaussian");
+  GaussianEmission *ge = std::get<GaussianEmission *>(ev);
```

Comment:
Could you run clang-format? Elsewhere we've written pointer types with spacing like `GaussianEmission* ge`, and clang-format should make that consistent (since our config file has `PointerAlignment: Left`).

Reply:
Done.